.github/pr-title-checker-config.json

```diff
@@ -5,18 +5,13 @@
   },
   "CHECKS": {
     "prefixes": [
-      "fix: ",
-      "feat: ",
-      "break: ",
-      "docs: ",
-      "chore: ",
-      "platform: "
+      "chore: "
     ],
-    "regexp": "(fix|feat|break|docs|chore|platform)\\((arm|bicep|bitbucket|circleci|cloudformation|dockerfile|github|gitlab|helm|kubernetes|kustomize|openapi|sca|secrets|serverless|terraform|general|graph)\\): "
+    "regexp": "(fix|feat|break|docs|chore|platform)\\((argo|arm|bicep|bitbucket|circleci|cloudformation|dockerfile|github|gitlab|helm|kubernetes|kustomize|openapi|sca|secrets|serverless|terraform|general|graph)\\): "
```

Comment:
Should we have a 1:1 map between frameworks to the regex?
If so, `github_actions` is missing (or should we use `github` instead?)

Reply:
I wanted it to keep a bit more simple for now, but if we feel like we want to separate it more, then sure lets do it in a later step 🙂